_inc/lib/debugger/class-jetpack-debugger.php

```diff
@@ -5,56 +5,31 @@
  * @package jetpack
  */
 
-/** Ensure the Jetpack_Debug_Data class is available. It should be via the library loaded, but defense is good. */
-require_once 'class-jetpack-debug-data.php';
-
 /**
  * Class Jetpack_Debugger
  *
  * A namespacing class for functionality related to the legacy in-plugin diagnostic tooling.
  */
 class Jetpack_Debugger {
-
-	/**
-	 * Determine the active plan and normalize it for the debugger results.
-	 *
-	 * @return string The plan slug prepended with "JetpackPlan"
-	 */
-	private static function what_jetpack_plan() {
-		// Specifically not deprecating this function since it modifies the output of the Jetpack_Debug_Data::what_jetpack_plan return.
```

Comment:
Is this okay to remove this now? The comment makes me worry. :)

Reply:
Yeah, that function provided some of the "debug data" that was part of the appended data when using the contact form that was built into Jetpack and all that debug data was legacy for WP versions prior to 5.2 (the old "Advanced Debug Results" link. The specific format of the text was required for Kayako/Zendesk processing to assign a plan value to a ticket.

For WP 5.2+, this stuff never displayed anywhere and with all private support requests going to the contact form (which uses the API and doesn't parse text anyhow), the plan output isn't needed here nor the legacy format that we needed to maintain.